packetbeat/protos/http/event.go

```diff
@@ -90,6 +90,12 @@ func newURL(host string, port int64, path, query string) *ecs.Url {
 	if port != 80 {
 		u.Port = port
 	}
+	if path != "" && strings.Contains(path, ".") {
+		periodIndex := strings.LastIndex(path, ".")
```

Comment:
The `Contains` and `LastIndex` operations are a bit redundant. To save a few cpu cycles, it could check the return of `LastIndex` for `-1`.

Reply:
will do

Reply:
done